go/ngql/query_test.go

```diff
@@ -1110,8 +1110,9 @@ func (suite *QueryGraphQLSuite) TestSetWithComplexKeys() {
 }
 
 func (suite *QueryGraphQLSuite) TestInputToNomsValue() {
+	var vr types.ValueReader
```

Comment:
can you just go ahead and create a test one here. Once my patch lands, it's going to be bad practice to ever hand in a null VRW.